spring-cloud-gcp-data-firestore/src/main/java/org/springframework/cloud/gcp/data/firestore/FirestoreTemplate.java

```diff
@@ -54,6 +55,12 @@
  */
 public class FirestoreTemplate implements FirestoreReactiveOperations {
 
+	private static final StructuredQuery.Projection ID_PROJECTION = StructuredQuery.Projection.newBuilder()
```

Comment:
I'd make `__name__` a public constant, at minimum in this class. Would it also make sense to have a class of public Firestore-related constants that's standalone? Then both reactive and non-reactive functionality can use the same basic constants.